e2e/scripts/run-service.sh

```diff
@@ -12,7 +12,7 @@ if [[ ! -z $LISK_SERVICE_FILE_PATH ]]
 
     echo LISK_APP_WS=ws://host.docker.internal:7887 >>${LISK_SERVICE_FILE_PATH}/.env
```

Comment:
NRTTL
I wasnt able to get the auto update to work. It didnt refresh the page after index update nor when indexing was complete.
@ikem-legend could you test this refresh part and see if it works for you? Might be something strange with my service setup 🤔

Reply:
@oskarleonard The refresh part worked for me